src/Language/GraphQL/Draft/Generator.hs

```diff
@@ -101,6 +103,43 @@ genObjectValue genVal = M.fromList <$> mkList genObjectField
   where
     genObjectField = (,) <$> genName <*> genVal
 
+genBlockText :: Gen Text
+genBlockText =
+  Gen.choice [ simple, genLines ]
+ where
+  simple = do
+    Gen.frequency
+      [ (10, Gen.text (Range.linear 1 100) Gen.unicode)
+      , (10, return "\n")
+      , (6, genIndentation)
+      , (5, genMinIndentedText 10)
+      , (4, return "")
+      , (3, return " ")
+      , (3, return "\"") -- "
+      , (3, return "\\") -- \
+      ]
+  genLines :: Gen Text
+  genLines = do
+    n <- Gen.int (Range.linear 1 100)
+    x <- Gen.list (Range.linear 1 n) simple
+    return (T.unlines x)
```

Comment:
This looks to me like it's only testing a non-zero number of lines, each of non-zero length - is that right? Can we include some `0`s in these options?